index.js

```diff
@@ -110,7 +115,7 @@ module.exports = (text, options) => {
 		lines = lines.concat(new Array(padding.bottom).fill(''));
 	}
 
-	const contentWidth = widestLine(text) + padding.left + padding.right;
+	const contentWidth = widestLine(options.title + '      \n' + text) + padding.left + padding.right;
```

Comment:
@sindresorhus You can see here that I verify the widest line of text and title. If title will be very long, then box width  will be increased.